lib/Extractor/KLEEBuilder.cpp

```diff
@@ -74,6 +74,10 @@ struct ExprBuilder {
   std::vector<Inst *> &ArrayVars;
   std::vector<Inst *> PhiInsts;
   UniqueNameSet ArrayNames;
+  // A flag that is used to control if the UBInst under process has
+  // any precondition, i.e., blockpc.
+  bool UBInstHasPrecondition;
+  ref<Expr> UBInstPrecondition;
```

Comment:
There is a [isNull()](https://github.com/klee/klee/blob/master/include/klee/util/Ref.h#L99) method.